docs/adr/001.md

```diff
@@ -45,7 +45,7 @@ Requirements for the `AmplitudeModel`:
    for the calculation of fit fractions, or plotting individual parts of the
    intensity.
 
-## Decision Drivers
+## Decision drivers
```

Comment:
Note the style change: titles are not capitalized anymore to avoid problems in the sidebar:
![image](https://user-images.githubusercontent.com/29308176/106450331-525a2700-6485-11eb-871d-9de2a19d66ba.png)

This could be read as:
- Contex and Problem
- Statement
- ...

but was actually
- Contex and Problem Statement
- ...